calicoctl/calico_ctl/node.py

```diff
@@ -847,8 +848,12 @@ def handle_sigterm(sig, frame):
     except KeyboardInterrupt:
         # Mainline. Someone pressed Ctrl-C.
         print "Stopping Calico node..."
+        stop_container_on_exit = True
+        exit_code = 130
     finally:
         # Could either be this process is being killed, or output generator
         # raises an exception.
         if stop_container_on_exit:
             docker_client.stop(container)
+            # If the container is stopped, some sort of error occurred.
+            sys.exit(exit_code)
```

Comment:
We don't?

`exit_code` is set to either 1 or 130 in this function, it will never be 0 if the container is stopped here.